app/controllers/orders_controller.rb

```diff
@@ -1,6 +1,7 @@
 class OrdersController < ApplicationController
   def show
     @order = Order.find(params[:id])
+    @items = @order.order_items
```

Comment:
Yeah, does seem like includes would work here, weird that @order.items wouldnt work...